test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -54,6 +54,11 @@ module ServiceManager
       end
 
       it 'should validate download url' do
+        @handler.build_download_url
+        assert_silent { @handler.validate_url }
```

Comment:
> assert_silent { @handler.validate_url }

Here, I am testing whether the download URL built from already provided options in `before` block is fine enough and should not throw any kind of exception against what it is expected to have some exception on invalid URL which is covered by the test case just below this test case.